pentaho/src/test/java/pt/webdetails/cpf/MimeTypeHandlerTest.java

```diff
@@ -31,69 +29,27 @@
   private static final String EXTENSION_UNKNOWN_BY_ALL = "qwerty";
   private static final String EXPECTED_OUTCOME_UNKNOWN_BY_ALL = "application/unknown";
 
-  private Log logger = LogFactory.getLog( MimeTypeHandlerTest.class );
-
   @Test
   public void testGetExtensionOnlyKnownInPlatform() {
 
-    String actualOutcome = StringUtils.EMPTY;
-
-    try {
-
-      actualOutcome = MimeTypeHandler.getMimeTypeFromExtension( EXTENSION_ONLY_KNOW_IN_PLATFORM );
+    String actualOutcome = MimeTypeHandler.getMimeTypeFromExtension( EXTENSION_ONLY_KNOW_IN_PLATFORM );
 
-      log( "testGetExtensionOnlyKnownInPlatform()", EXTENSION_ONLY_KNOW_IN_PLATFORM, EXPECTED_OUTCOME_IN_PLATFORM,
-        actualOutcome );
-
-    } catch ( Throwable t ) {
-      logger.error( t );
-      Assert.fail();
-    }
-
-    Assert.assertTrue( EXPECTED_OUTCOME_IN_PLATFORM.equalsIgnoreCase( actualOutcome ) );
+    assertTrue( EXPECTED_OUTCOME_IN_PLATFORM.equalsIgnoreCase( actualOutcome ) );
   }
 
   @Test
   public void testGetExtensionOnlyKnownInCpf() {
 
-    String actualOutcome = StringUtils.EMPTY;
-
-    try {
-
-      actualOutcome = MimeTypeHandler.getMimeTypeFromExtension( EXTENSION_ONLY_KNOW_IN_CPF );
+    String actualOutcome = MimeTypeHandler.getMimeTypeFromExtension( EXTENSION_ONLY_KNOW_IN_CPF );
 
-      log( "testGetExtensionOnlyKnownInCpf()", EXTENSION_ONLY_KNOW_IN_CPF, EXPECTED_OUTCOME_IN_CPF, actualOutcome );
-
-    } catch ( Throwable t ) {
-      logger.error( t );
-      Assert.fail();
-    }
-
-    Assert.assertTrue( EXPECTED_OUTCOME_IN_CPF.equalsIgnoreCase( actualOutcome ) );
+    assertTrue( EXPECTED_OUTCOME_IN_CPF.equalsIgnoreCase( actualOutcome ) );
   }
 
   @Test
   public void testGetExtensionUnknownByAll() {
 
-    String actualOutcome = StringUtils.EMPTY;
-
-    try {
-
-      actualOutcome = MimeTypeHandler.getMimeTypeFromExtension( EXTENSION_UNKNOWN_BY_ALL );
-
-      log( "testGetExtensionUnknownByAll()", EXTENSION_UNKNOWN_BY_ALL, EXPECTED_OUTCOME_UNKNOWN_BY_ALL,
-        actualOutcome );
-
-    } catch ( Throwable t ) {
-      logger.error( t );
-      Assert.fail();
-    }
-
-    Assert.assertTrue( EXPECTED_OUTCOME_UNKNOWN_BY_ALL.equalsIgnoreCase( actualOutcome ) );
-  }
+    String actualOutcome = MimeTypeHandler.getMimeTypeFromExtension( EXTENSION_UNKNOWN_BY_ALL );
 
-  private void log( String methodName, String testExtension, String expectedOutcome, String actualOutcome ) {
```

Comment:
why did you remove this log method? (related with the previous questions)

Reply:
It's no longer referred.